parsec/core/types/backend_address.py

```diff
@@ -137,7 +160,12 @@ def use_ssl(self):
         return self._use_ssl
 
 
-class OrganizationParamsFixture(BackendAddr):
+class BackendOrganizationAddr(BackendAddr):
+    """
+    Represent the URL to access an organization within a backend
+    (e.g. ``parsec://parsec.example.com/MyOrg?rvk=7NFDS4VQLP3XPCMTSEN34ZOXKGGIMTY2W2JI2SPIHB2P3M6K4YWAssss``)
```

Comment:
Is this really a valid URL? I doesn't contain any action to perform.

Reply:
only `BackendActionAddr` and children contain action parameter